packages/web-sdk/src/transports/fetch/transport.ts

```diff
@@ -60,9 +60,14 @@ export class FetchTransport extends BaseTransport {
               this.disabledUntil = this.getRetryAfterDate(response);
               this.logWarn(`Too many requests, backing off until ${this.disabledUntil}`);
             }
-
+            // read the body so the connection can be closed
+            response
+              .text()
+              .then(() => {})
```

Comment:
Just a nitpick: I think we can use `noop` here

Reply:
removed the `.then()`, I think it's redundant, and added `noop` to `.catch()`